src/apis/queryKeys.ts

```diff
@@ -9,7 +9,8 @@ const queryKey = {
     all: ['posts'] as const,
     detail: (id: string) => ['posts', id] as const,
     search: (userId: string) => ['posts', 'search', userId] as const
-  }
+  },
+  comments: ['comments'] as const
```

Comment:
P2:
나중에 특정 ID의 코멘트 상세 정보만 관리하는 쿼리 키를 만들어야 한다면, 수정해야 할 부분이 많아질 것 같아요!

```suggestion
  comments: {
    all: ['comments'] as const
  }
```

요런 느낌으로 미리 확장성에 대해서 열어두는건 어떨까요?

Reply:
구현 초반에 만들어 놓았던 것 같은데, 지금 보니 comment query key가 안 쓰이고 있는 것 같아서 삭제해도 괜찮지 않을까 싶네요.
comment가 따로 api로 받아올 수가 없고 post에 종속되어 있다보니 앞으로도 못 쓰는게 맞겠죠? 🤔